container/functional/keylime_agent_container-basic-attestation/test.sh

```diff
@@ -8,6 +8,8 @@
 
 [ -n "$AGENT_DOCKERFILE" ] || AGENT_DOCKERFILE=Dockerfile.upstream.c9s
 
+[ -n "$REGISTRY" ] || REGISTRY=quay.io
```

Comment:
Should we make images configurable through envvars as well?

Reply:
Now I can see we do configure it through envvars, it's just not clearly visible in the test. What about adding a comment to the top?